docs/cuda-miner.md

```diff
@@ -0,0 +1,104 @@
+# CUDA Miner
+
+As the release packages do not ship with CUDA miner, one should build it themselves. It consists of the below steps:
```

Comment:
Perhaps:
"The release packages do not ship with a CUDA miner, but you can build it yourself by following these steps:"